CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 cmake_minimum_required (VERSION 3.6)
 project(Radium-apps)
-option(RADIUM_APP_FETCH_RADIUM "Fetch, compile and link to a local copy of Radium" OFF)
```

Comment:
maybe we need this one and it's a rebase error ?
@nmellado could you confirm before merge ?

Reply:
It’s ok to remove the option, the functionality has not been implemented